SPECS/nexus-oss.spec

```diff
@@ -101,6 +111,8 @@ fi
 %postun
 %if %use_systemd
 /usr/bin/systemctl daemon-reload
+%else
+/sbin/chkconfig --add %{name}
```

Comment:
Why do we need ```chkconfig -add``` on post uninstallation? The service should is already removed from chkconfig on on ```%preun``` (I did it there, since it must be removed BEFORE ```/etc/init.d/nexus is removed```